lib/raster/mask_info.c

```diff
@@ -49,6 +49,27 @@ char *Rast_mask_info(void)
     return G_store(text);
 }
 
+/**
+ * @brief Retrieves the name of the raster mask to use.
+ *
+ * The returned raster map name is fully qualified, i.e., in the form
+ % "name@mapset".
```

Comment:
What does the % mean here instead of a *?

Reply:
That's a typo. I can fix it in #2392.

Reply:
For the record: Fixed in 4465c562f3c0b7ab58f12a588aa0f47890f271a2